addon/helpers/object-at.js

```diff
@@ -15,22 +11,6 @@ export function objectAt(index, array) {
   return emberArray(array).objectAt(index);
```

Comment:
for a future PR: we can likely improve performance by, instead of converting to an `emberArray`, checking if the array is already and `emberArray` and using `objectAt` or, if it's a native array, using `array[index]`